.travis.yml

```diff
@@ -12,7 +12,6 @@ branches:
     - master
 
 env:
-  - TARGET="56"
```

Comment:
Ok got it, I tried executing tests with new deps and php5.6

first error : 
```
Problem 1
    - This package requires php ^7.0 but your PHP version (5.6.31) does not satisfy that requirement.
  Problem 2
    - Installation request for elasticsearch/elasticsearch dev-master -> satisfiable by elasticsearch/elasticsearch[dev-master].
    - elasticsearch/elasticsearch dev-master requires ext-json >=1.3.7 -> the requested PHP extension json has the wrong version (1.2.1) installed.
```

due to composer.json reqs in es-php, so I try using **--ignore-platform-reqs** in order to skip php platform requirements.

Then we have another error :

> Parse error: syntax error, unexpected ':', expecting ';' or '{' in /root/composer/vendor/doctrine/instantiator/src/Doctrine/Instantiator/Instantiator.php on line 95
> Call Stack:
>     0.0002     234056   1. {main}() /root/composer/vendor/phpunit/phpunit/phpunit:0
>     0.0184    1065728   2. PHPUnit_TextUI_Command::main() /root/composer/vendor/phpunit/phpunit/phpunit:52
>     0.0185    1066480   3. PHPUnit_TextUI_Command->run() /root/composer/vendor/phpunit/phpunit/src/TextUI/Command.php:116
>     0.9737   32252296   4. PHPUnit_TextUI_TestRunner->doRun() /root/composer/vendor/phpunit/phpunit/src/TextUI/Command.php:186
>     1.0118   32801632   5. PHPUnit_Framework_TestSuite->run() /root/composer/vendor/phpunit/phpunit/src/TextUI/TestRunner.php:517
>     2.2593   40272824   6. PHPUnit_Framework_TestSuite->run() /root/composer/vendor/phpunit/phpunit/src/Framework/TestSuite.php:722
>     2.2648   40276728   7. PHPUnit_Framework_TestSuite->run() /root/composer/vendor/phpunit/phpunit/src/Framework/TestSuite.php:722
>     2.2649   40281032   8. PHPUnit_Framework_TestCase->run() /root/composer/vendor/phpunit/phpunit/src/Framework/TestSuite.php:722
>     2.2649   40281656   9. PHPUnit_Framework_TestResult->run() /root/composer/vendor/phpunit/phpunit/src/Framework/TestCase.php:860
>     2.2663   40284560  10. PHPUnit_Framework_TestCase->runBare() /root/composer/vendor/phpunit/phpunit/src/Framework/TestResult.php:686
>     2.2680   40302592  11. PHPUnit_Framework_TestCase->runTest() /root/composer/vendor/phpunit/phpunit/src/Framework/TestCase.php:905
>     2.2690   40303744  12. ReflectionMethod->invokeArgs() /root/composer/vendor/phpunit/phpunit/src/Framework/TestCase.php:1054
>     2.2690   40303848  13. Elastica\Test\Bulk\ResponseSetTest->testIsOk() /root/composer/vendor/phpunit/phpunit/src/Framework/TestCase.php:1054
>     2.2690   40304024  14. Elastica\Test\Bulk\ResponseSetTest->_createResponseSet() /elastica/test/Elastica/Bulk/ResponseSetTest.php:21
>     2.2691   40304184  15. PHPUnit_Framework_TestCase->createMock() /elastica/test/Elastica/Bulk/ResponseSetTest.php:145
>     2.2725   40587416  16. PHPUnit_Framework_MockObject_MockBuilder->getMock() /root/composer/vendor/phpunit/phpunit/src/Framework/TestCase.php:1460
>     2.2726   40587792  17. PHPUnit_Framework_MockObject_Generator->getMock() /root/composer/vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/MockBuilder.php:118
>     2.3138   40756808  18. PHPUnit_Framework_MockObject_Generator->getObject() /root/composer/vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/Generator.php:275
>     2.3158   40954640  19. spl_autoload_call() /root/composer/vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/Generator.php:305
>     2.3160   40954880  20. Composer\Autoload\ClassLoader->loadClass() /root/composer/vendor/phpunit/phpunit-mock-objects/src/Framework/MockObject/Generator.php:305
>     2.3163   40955064  21. Composer\Autoload\includeFile() /root/composer/vendor/composer/ClassLoader.php:322



any idea anyone ? maybe it's an easier task not but I'm nop sure about it.
making a quick search it depends on [this issue](https://github.com/doctrine/instantiator/issues/30) 
@ruflin would you like from me to get a little more deeper ?